apps/web/components/Organization/Wrapper/Themes/TryggingastofnunTheme/TryggingastofnunHeader.tsx

```diff
@@ -22,6 +23,14 @@ const TryggingastofnunHeader: React.FC<
   )
   const n = useNamespace(namespace)
 
+  const { activeLocale } = useI18n()
+  const logoAltText = n(
```

Comment:
Just in case there isn't a title for the organization perhaps it'd be better to fallback to an empty string instead of potentially concatenating undefined or '' with ' Frontpage'